web/assets/css/transmission-app.scss

```diff
@@ -962,16 +958,6 @@ $video-image: '../img/film.svg';
   }
 }
 
-.tabs-container-close {
-  font-size: 150%;
-  cursor: pointer;
-  background: var(--color-bg-primary);
-  border: 0;
-  color: var(--color-fg-primary);
-  position: absolute;
-  padding: 10px 16px;
-}
-
```

Comment:
Though I have to ask, why was this deleted as well?

Reply:
That was an error on my part, but until we see which way to go, i'm not going to update anything.